src/app/layout.tsx

```diff
@@ -1,26 +1,33 @@
 // layout.tsx
 import type { Metadata } from "next";
-import React from 'react';
-import ClientLayout from './ClientLayout';
 import { Barlow } from "next/font/google";
+import React from "react";
+import { PopupProvider } from "./_contexts/Popup.context";
 import "./_utils/s3Utils";
+import ClientLayout from "./ClientLayout";
 
 const barlow = Barlow({
-	subsets: ["latin"],
-	weight: ["400", "600", "800"],
-	variable: "--font-barlow",
+  subsets: ["latin"],
+  weight: ["400", "600", "800"],
+  variable: "--font-barlow",
 });
 
 export const metadata: Metadata = {
-	title: "Karabast",
+  title: "Karabast",
 };
 
-export default function RootLayout({ children }: { children: React.ReactNode }) {
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
   return (
     <html lang="en" className={barlow.variable}>
       <body>
-        <ClientLayout>{children}</ClientLayout>
+        <PopupProvider>
```

Comment:
Lets include this with the other client providers in `_sharedcomponents/ClientProviders` since this is likely somethining we'll want app wide.